### ISSUE TITLE
pygobject (bottled): missing shared library _glib.so 

### ISSUE THREAD

**Issue by Char-Aznable** [Linuxbrew/homebrew-core#5439]
# Please fill out the issue checklist below, and provide all of the requested information.

# Please always follow these steps:
- [x] Confirmed this is a problem with `brew install`ing one, specific formula and not every time you run `brew`? If it's a general `brew` problem please file this issue at https://github.com/Linuxbrew/brew/issues/new.
- [ ] If you have a macOS system and can replicate the problem on macOS with Homebrew, please report the issue to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and follow their issue template from your macOS system. If you are unsure, please report the issue to Linuxbrew.
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [ ] Ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
- [ ] If `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
The bottled pyobject 2.28.7 lacks the shared library _glib.so, which breaks diffuse:
```
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/diffuse", line 126, in <module>
    import gtk
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python2.7/site-packages/gtk-2.0/gtk/__init__.py", line 30, in <module>
    import gobject as _gobject
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python2.7/site-packages/gobject/__init__.py", line 26, in <module>
    from glib import spawn_async, idle_add, timeout_add, timeout_add_seconds, \
  File "/home/linuxbrew/.linuxbrew/opt/python/lib/python2.7/site-packages/glib/__init__.py", line 22, in <module>
    from glib._glib import *
ImportError: No module named _glib
```
Listing the files in /home/linuxbrew/.linuxbrew/opt/python/lib/python2.7/site-packages/glib:
```
option.pyc
__init__.py -> ../../../../Cellar/pygobject/2.28.7/lib/python2.7/site-packages/glib/__init__.py
_glib.a -> ../../../../Cellar/pygobject/2.28.7/lib/python2.7/site-packages/glib/_glib.a
option.py -> ../../../../Cellar/pygobject/2.28.7/lib/python2.7/site-packages/glib/option.py
__init__.pyc
```
and files in /home/linuxbrew/.linuxbrew/Cellar/pygobject/2.28.7/lib/python2.7/site-packages/glib/
```
option.py
__init__.py
_glib.a
```

Only the static library _glib.a is found but not the _glib.so.

Doing
```
brew reinstall pygobject --build-from-source
```
fixes the issue with files in /home/linuxbrew/.linuxbrew/opt/python/lib/python2.7/site-packages/glib
```
__init__.py -> ../../../../Cellar/pygobject/2.28.7/lib/python2.7/site-packages/glib/__init__.py
_glib.so -> ../../../../Cellar/pygobject/2.28.7/lib/python2.7/site-packages/glib/_glib.so
option.py -> ../../../../Cellar/pygobject/2.28.7/lib/python2.7/site-packages/glib/option.py
__init__.pyc
option.pyc
```

**Comment by iMichka** [comment]
Hi. Thanks for reporting this. We had some bottles which did not build correctly at one point, but I could not make the exact list.

Could you open a pull request for this? Remove the Linux sha256, and bump the revision, so that a new bottle gets built.

**Comment by Char-Aznable** [comment]
PR #5447 submitted

**Comment by iMichka** [comment]
Fixed by #5447
